Neos.Flow/Classes/ObjectManagement/Configuration/Configuration.php

```diff
@@ -191,6 +191,9 @@ public function getPackageKey()
     public function setFactoryObjectName($objectName)
     {
         $this->factoryObjectName = $objectName;
+        if ($this->factoryMethodName === '') {
+            $this->factoryMethodName = 'create';
```

Comment:
This seems weird, why is this change required?

Reply:
Because `factoryMethodName` was sets with a default value in the property declaration, which led to ALL classes being handled as if they should be created by factory.
`Error: Method "create" does not exist.`